cub/cub/detail/nvtx.cuh

```diff
@@ -41,7 +41,7 @@
 // * The NVTX3 C API is available in CTK
 // * NVTX is not explicitly disabled
 // * C++14 is availabl for cuda::std::optional
-#if __has_include(<nvtx3/nvToolsExt.h> ) && !defined(NVTX_DISABLE) && _CCCL_STD_VER >= 2014
```

Comment:
Please add a comment that nvc++ at C++20 is disabled because the nvtx3 headers use the name `module` as a parameter name, which is a keywoard in C++20.